proto/connectrpc/conformance/v1/service.proto

```diff
@@ -316,43 +318,53 @@ message ConformancePayload {
     ConnectGetInfo connect_get_info = 4;
   }
   message ConnectGetInfo {
+    // The query params observed in the request URL.
     repeated Header query_params = 1;
   }
 }
 
 // An error definition used for specifying a desired error response
 message Error {
+  // The error code.
```

Comment:
I would link to relevant docs:
https://connectrpc.com/docs/protocol#error-codes